docs/release_checklist.md

```diff
@@ -1,9 +1,17 @@
 # Release checklist
 
-When changing the version, run `build-scripts/update-version.sh X Y Z`,
-where `X Y Z` are the major version, minor version, and patch level. So
-`3 8 1` means "change the version to 3.8.1". This script does much of the
-mechanical work.
+## Scripts
```

Comment:
This generally looks good, but why do we have a subheading here?

Reply:
I thought the tools deserved their own section.

It's removed now.